src/status_im/common/keychain/events.cljs

```diff
@@ -131,6 +130,13 @@
    (keychain/reset-credentials (str key-uid "-auth"))
    (keychain/reset-credentials key-uid)))
 
+(re-frame/reg-fx
+ :keychain/clear-keycard-keys
+ (fn [key-uid]
+   (keychain/reset-credentials (str key-uid "-auth"))
```

Comment:
Could we extract the `(str key-uid "-auth")` to its own function too?
Not sure what to name that function, but would `auth-key-name` be okay?

Reply:
this namespace is legacy i don't want to change it much, it should be refactored

Reply:
done